src/test/groovy/pl/noip/lolstats/lol/stats/service/RiotRestClientTest.groovy

```diff
@@ -0,0 +1,24 @@
+package pl.noip.lolstats.lol.stats.service
+
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.test.context.ActiveProfiles
+import spock.lang.Specification
+
+
+@SpringBootTest
+@ActiveProfiles("test")
+class RiotRestClientTest extends Specification {
+
+    @Autowired
+    RiotRestClient riotRestClient
+
+    def "CheckUserNameInRiotDataBase"() {
```

Comment:
@Ignore this test.
Add comment to this test, that it requires API key which lasts for one day.